### ISSUE TITLE
Sound notification does not work when running with cron

### ISSUE THREAD

**Issue by BurnyBoi** [LukeSmithxyz/mutt-wizard#111]
I currently have mailsync.sh set to run in cron every 5 minutes. It's able to run offlineimap, pull in new messages, and have the notification pop up on my screen just fine. The only part that isn't working is the line that is supposed to play notify.opus. I have mpv installed, and if I run mailsync.sh either from the terminal, or in neomutt by pressing 'o', the sound notification works as expected. 

Any ideas on how to troubleshoot the sound not working when running from cron? Here is my current crontab:

DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
DISPLAY=:0
*/5 * * * * ~/.config/mutt/etc/mailsync.sh

(note: I've tried it without the first 2 lines, and still wasn't working)

Running this on Parabola. Any info you have on this would be much appreciated.

**Comment by chswb** [comment]
This is just a guess but it could be that your `PATH` variable is set differently for cron. If `notify-send` and `mpv` are then in different directories and `mpv` is not in the `PATH` variable in cron then `mpv` would not work even though the notification still does.

Try comparing the two `PATH` variables to see if they're the same.

**Comment by BurnyBoi** [comment]
Thank you for the suggestion. I wasn't explicitly setting PATH in my crontab, so I looked up how to do so here:
https://stackoverflow.com/questions/2388087/how-to-get-cron-to-call-in-the-correct-paths

I used crontab -e, and added my PATH so that it my crontab looks like this:
PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus
DISPLAY=:0
*/5 * * * * ~/.config/mutt/etc/mailsync.sh

Even with this here, I'm still not hearing the sound notification when new mail comes in. As a further test, I also added this to the end of my crontab:
*/1 * * * * notify-send HI && mpv ~/.config/mutt/etc/notify.opus 

This is just to test if there is possibly anything in the script that could be to blame. With this test cronjob, I see the notification "HI", but still no sound.

**Comment by BurnyBoi** [comment]
Okay, I think I finally got it. I stumbled across this:
https://askubuntu.com/questions/1045269/17-10-no-sound-from-cron-jobs

They mentioned I needed to set XDG_RUNTIME_DIR. So I added the following to the top of my crontab:
XDG_RUNTIME_DIR="/run/user/1000"

With this, it's finally working! Sorry for my triple post here, but just wanted to share my solution. Thanks again chswb for having a look. Closing this out.
